platform/node/bitrise.yml

```diff
@@ -57,11 +61,15 @@ workflows:
             set -eu -o pipefail
             brew update
             brew unlink node
-            brew install cmake awscli node@4
+            brew install cmake awscli node@4 node@6
             brew link node@4 --force
             gem install xcpretty --no-rdoc --no-ri
             export BUILDTYPE=Release
             export PUBLISH=true
             make test-node
+            brew unlink node@4
+            brew link --overwrite node@6 --force
+            make clean
+            make test-node
             ./platform/node/scripts/after_success.sh
```

Comment:
`after_success.sh` is what performs the publish, so it needs to be run after both the node@4 and node@6 builds or otherwise only the latter will get published. Suggest something like:

```
make test-node && ./platform/node/scripts/after_success.sh
```